### ISSUE TITLE
Nation zones are "protecting" towns even if they aren't in the nation.

### ISSUE THREAD

**Issue by Momshroom** [TownyAdvanced/Towny#3029]
This ticket template is for bug/error reports. If you are suggesting a feature and not reporting a bug please delete all the text here and begin your ticket Title with Suggestion:

### What steps will reproduce the problem?
1. Enable nation zones around towns in the config
2. make a nation.
3. have some towns join the nation and some not.
4. Even the towns that are not in the nation and NOT near any towns in the nation will get nation zones around them from the nation and be unable to use the wilds around their own towns. (When they try they're told the area is protected by the nation)

### What is the expected output?
1. Nation zones are expected only around that nation's towns.
2.
3.

### What is your Towny version?
Do not say "latest".
http://prntscr.com/krgsdp
last posted dev build of towny from the site
0.92.0.14

### What is your spigot/craftbukkit version?
Do not say "latest".
1.13.1 (exact version in screenshot above)

### Please use Pastebin.com to link the following files:

**Skipping these files is not an option. It is the easiest way to diagnose an issue ticket properly. If you do skip this step then your ticket will most likely be deleted and you will be asked to resubmit.**
1. Your full server startup from the logs\latest.log :
https://pastebin.com/wL1KGWba
2. Your Towny config.yml (if using MySQL please remove password,) :
https://pastebin.com/ktvgWHzM
3. Your townyperms.yml :
https://pastebin.com/0az4y7ms
4. Your log's error : 
No console error
(optional) If this is to do with permissions, 
5. Your permissions file :

(optional) If this is to do with chat,
6. Your chatconfig.yml :
7. Your channels.yml :

**Comment by Momshroom** [comment]
Still working on this.. didn't mean to submit it yet. the rest of the files are coming.

**Comment by LlmDl** [comment]
If you don't mind, can you quickly test if it still happens when only_capitals: 'true' is set instead of false?

**Comment by Momshroom** [comment]
sure.

**Comment by Momshroom** [comment]
It doesn't happen if only capitals is set true.

**Comment by LlmDl** [comment]
Can you try testing it with this build for me: https://www.dropbox.com/s/2yqmudqllj6mk96/Towny%20momshroom.jar?dl=0

**Comment by Momshroom** [comment]
That seems to have fixed it. Tyvm.

**Comment by LlmDl** [comment]
Thanks for finding that one.

**Comment by Momshroom** [comment]
Sorry I reported prematurely.   I’m getting reports that it’s still broken.

**Comment by LlmDl** [comment]
Any word on the way in which it is broken?

**Comment by Momshroom** [comment]
Nationless towns still have a nation zone they can't use.

**Comment by Momshroom** [comment]
Yes, I've just confirmed with one of my staff members testing.  Nationless towns still have nation zones.

**Comment by LlmDl** [comment]
Huh, well I'll go back to the drawing board and test it on my own until I get this right.

**Comment by LlmDl** [comment]
I take it back, we were only checking for nearest towns with a nation, try this build for me: 
https://www.dropbox.com/s/fqj63ixu2c6gam2/Towny%20momshroom2.jar?dl=0

**Comment by Momshroom** [comment]
That appears to fix it.  Ty.
